modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
+
+The link:https://github.com/openshift/console-plugin-template[plugin template] demonstrates how you can translate messages with link:https://www.i18next.com/[react-i18next].
+
+.Prerequisites
+* You must have the plugin template cloned locally.
+* Optional: To test your plugin locally, run the {product-title} web console in a container. You can use either Docker or Podman 3.2.0+.
+
+.Procedure
+
+. Prefix the name with `plugin\__` to avoid any naming conflicts. The plugin template uses the `plugin__console-plugin-template` namespace by default, and you must update when you rename your plugin for example, `plugin__my-plugin`.  You can use the `useTranslation` hook, for example:
++
+[source,tsx,subs="+quotes,+macros"]
+----
+conster Header: React.FC = () ++=>++ {
+  const ++{ t } = useTranslation('plugin__console-demo-plugin');++
+  ++return <h1>{t('Hello, World!')}</h1>;++
+++};++
+----
++
+[IMPORTANT]
+====
+You must match the `i18n` namespace with the name of the `ConsolePlugin` resource.
+====
+
+. Set the `spec.i18n.loadType` field based on needed behavior. `Preload` will load all plugin's localization resources from the i18n namespace after the dynamic plugin during loading. For example, `plugin__console-demo-plugin`
```

Comment:
- Consider defining `loadType: Preload` after the block with a callout rather than in the step text. 
- Could likewise make the "For example, plugin__console-demo-plugin" fragment into a title for the block
- Also present tense :)

```suggestion
. Set the `spec.i18n.loadType` field based on needed behavior. The `Preload` setting loads all the plugin's localization resources from the `i18n` namespace after the dynamic plugin during loading. For example, `plugin__console-demo-plugin`
```